src/qibolab/serialize.py

```diff
@@ -68,15 +68,22 @@ def load_qubits(
             for c, char in runcard["characterization"]["coupler"].items()
         }
 
-        for c, pair in runcard["topology"].items():
-            q0, q1 = pair
+        for pair, char in zip(
+            runcard["topology"].items(),
+            runcard["characterization"]["two_qubit"].values(),
+        ):
+            q0, q1 = pair[1]
             pairs[(q0, q1)] = pairs[(q1, q0)] = QubitPair(
```

Comment:
Is there a serious reason to use `zip` here? I think I would prefer to leave it as it is and create the pairs using `runcard["topology"]` only, and only add the characterization parameters if they are available. 
This should be possible with something like:
```suggestion
        for c, pair, char in runcard["topology"].items():
            q0, q1 = pair
            char = runcard["caracterization"].get("two_qubit", {}).get(pair, {})
            pairs[(q0, q1)] = pairs[(q1, q0)] = QubitPair(qubits[q0], qubits[q1], **char, coupler=couplers[json.loads(c)])
```
(but I haven't tested).

The main issue with the zip is that it requires the characterization section to contain the `"two_qubit"` key and all possible two-qubit pairs under it, and therefore breaks compatibility with existing platforms. I would avoid that hustle if possible and I think in this case we can avoid.